api/src/main/java/org/cloudfoundry/autoscaler/api/filter/AuthenticationFilter.java

```diff
@@ -55,7 +56,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             String authorization = httpReq.getHeader("Authorization");
             if ( authorization == null){
             	logger.info("No Authorization header found.");
-                httpResp.sendError(HttpServletResponse.SC_UNAUTHORIZED);
+                this.setResponse(httpResp, HttpServletResponse.SC_UNAUTHORIZED, MessageUtil.UNAUTHORIZED);
```

Comment:
This is not how MessageUtil works.
If you are going to add keys to MessageUtil, there should be a corresponding message added to the English NLS resource bundle.
You should be using the MessageUtil.getMessageXXXX method to load the appropriate message.